_posts/2015-11-24-hoodie-updates.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Hoodie 1.0 milestones closing in"
```

Comment:
The title doesn't read well to me. "Hoodie 1.0 Milestones: Closing In" appears better to my eyes, or something like that?